src/main/scala/pl/edu/agh/iet/akka_debugging/sbt/TracedActorsFinder.scala

```diff
@@ -0,0 +1,32 @@
+package pl.edu.agh.iet.akka_debugging.sbt
+
+import java.io.File
+
+import pl.edu.agh.iet.akka_debugging.sbt.ScalaClassDefinitionParser.parseClassDefinitions
+
+import scala.collection.mutable
+import scala.io.Source
+
+object TracedActorsFinder {
+  val TRAIT_NAME = "TracedActor"
```

Comment:
Well - I didn't know about this convention; it's strange that they have something very different from Java (the Java convention is used here now).

I'll correct it later.